docs/topics/native/native-ios-integration.md

```diff
@@ -48,9 +48,14 @@ The resulting output:
 ```text
 init on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 shared.SwiftExample
-deinit on <NSThread: 0x600003b9b900>{number = 7, name = (null)}
+deinit on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 ```
 
+* If Swift/Objective-C objects are passed to Kotlin on a thread other than main or if the main dispatch queue isn't processed,
```

Comment:
Maybe this bullet should be a list?
Something like:

Kotlin calls deinit on the S/O objects on a specific GC thread, if:
* S/O objects are passed to Kotlin on a thread other than the main
* Main dispatch queue isn't processed (by the GC?)

If you want to call deinit on .... explicitly, set ` ` in your `gradle.properties`. It calls deinit, even if these objects are passed to Kotlin on the main thread.